lib/post_data.js

```diff
@@ -132,9 +132,14 @@ MattermostDataPostHandler.prototype.postData = function(data) {
       content.text = chunks[i];
       content.fallback = chunks[i];
 
+      /*
+        Based on the issue Support for "Button attachments" #151
+        Inorder to accept the matteruser adapter message attachments changed the attachement json
+      */
+
       attachment = {
         room: recipient,
-        attachments: content,
+        content.attachments ? content.attachments : content,
```

Comment:
It's expected to have a dict here.

I believe the correct construction would be:
```js
attachments: content.attachments ? content.attachments: content,
```